zerver/views/streams.py

```diff
@@ -529,6 +543,80 @@ def you_were_just_subscribed_message(
     return message
 
 
+def you_were_just_unsubscribed_message(
```

Comment:
`get_just_unsubscribed_message_content` would be a clearer name for this, since it returns a string (the content), not a Message object.